client/app/kubernetes/translations/Messages_en_GB.xml

```diff
@@ -31,6 +31,9 @@
    <translation id="kube_service_billing_contact" qtlid="505008">Billing NIC handle</translation>
    <translation id="kube_service_billing_renewal" qtlid="168172">Renewal</translation>
 
+   <translation id="kube_service_reset_title">Reset this service</translation>
+   <translation id="kube_service_reset_message">You are about to put your service in the original state it was delivered on. This will delete all your nodes, your services, their configuration... <strong>This action is cannot be undone</strong>.</translation>
```

Comment:
```suggestion
   <translation id="kube_service_reset_message">You are about to put your service in the original state it was delivered on. This will delete all your nodes, your services, their configuration... <strong>This action cannot be undone</strong>.</translation>
```